modules/client/src/main/java/org/apache/ignite/internal/client/SslConfigurationImpl.java

```diff
@@ -72,6 +76,11 @@ public ClientAuthenticationMode clientAuthenticationMode() {
         return clientAuth;
     }
 
+    @Override
```

Comment:
/** {@inheritDoc} */

Reply:
Is it really necessary to add inheritDoc everywhere? I think it works just fine automatically if I override the documented method.